Source/MTBBlockedMessage.m

```diff
@@ -111,19 +119,24 @@ - (NSDictionary*)getTrackerDict {
             @"amazonappservices.com/r/",
             @"awscloud.com/trk"
         ],
+        @"Apo.com": @[@"info.apo.com/op/\\d+/.+.gif"],
         @"Apple": @[
           @"apple.com/report/2/its_mail_sf",
           @"apple_email_link/spacer",
         ],
         @"Appriver": @[@"appriver.com/e1t/o/"],
         @"Apptivo": @[@"apptivo.com"],
         @"Artegic": @[@"elaine-asp.de"],
+        @"Asana": @[@"app.asana.com/-/open"],
         @"ASUSTeK": @[@"emditpison.asus.com"],
         @"Atlassian": @[@"bitbucket.org/account/notifications/mark-read/"],
+        @"Autodesk": @["click.autodesk.com/trk"],
```

Comment:
Just FYI, this NSString is missing the `@` character. My automated test wasn't working properly 😅. I fixed this for next time

Reply:
Damn it - missed that, sorry 😅